### ISSUE TITLE
new Backups: IP/NIC allocation is not cleared on restore

### ISSUE THREAD

**Issue by atodorov-storpool** [OpenNebula/one#6046]
**Description**
When restoring a VM backup from the Sunstone interface there is an option to not restore NIC and IP attributes via checkboxes. It seems they are not handled.

**To Reproduce**
Create a VM backup
Restore to the backup VM template, selecting to not include NIC and IP (checked checkboxes)
Instantiate a VM from the backup template

Instantiation is failing with error  
> [one.template.instantiate] Error allocating a new virtual machine template. Cannot get IP/NIC lease from virtual network 0.

**Expected behavior**
A new VM should be instantiated

**Details**
 - Affected Component: [Sunstone]
 - Hypervisor: [KVM]
 - Version: [6.5.80]

**Additional context**
The error is because the initial VM is not terminated, but without networking it is expected the new VM to be instantiated with another IP...

The backup VM template still has NIC definition:

```
NIC = [
  FILTER_IP_SPOOFING = "YES",
  FILTER_MAC_SPOOFING = "YES",
  GATEWAY = "192.168.122.1",
  IP = "192.168.122.2",
  MODEL = "virtio",
  NETWORK = "192.168.122-virbr0",
  NETWORK_UNAME = "oneadmin",
  SECURITY_GROUPS = "0" ]
NIC_DEFAULT = [
  MODEL = "virtio" ]
```

## Progress Status
- [ ] Code committed
- [ ] Testing - QA
- [ ] Documentation (Release notes - resolved issues, compatibility, known issues)

**Comment by atodorov-storpool** [comment]
Edited to clarify: it is not MAC-related. The MAC issue is different (will open a separate issue for it)

**Comment by rsmontero** [comment]
Fixed by  595d73ba32209ed42c3e068e72991543514800dc

THANKS for the feedback :heart:
